tally-up-backend/template.yaml

```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  tally-up-backend
+Transform:
+  - AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    Runtime: nodejs22.x
+    Architectures:
+      - x86_64
+    MemorySize: 128
+    Timeout: 100
+    CodeUri: ./ # Not the function source code, but the location of the Makefile
+
+Resources:
```

Comment:
i understand its a yaml file im not understanding lie "who uses it and kinda where"

Reply:
in the dockerfile? in the aws stuffs? just for understanding not like request changes moment at this time.

Reply:
template.yaml is for AWS SAM which is a IAC tool like terraform